rules/autocorrect-in-outlook

```diff
@@ -0,0 +1,42 @@
+type: rule
+title: Do you know how to use Autocorrect in Outlook to insert common links you often send?
+uri: /rules/autocorrect-in-outlook
+authors:
+  - title: Ulysses Maclaren
+    url: https://www.ssw.com.au/people/uly
+created: 2024-01-08T13:42:55.753Z
+guid: 7a8b9c0d-3b2e-4f31-af5e-9f9f4a2d9d86
+---
+# Do you know how to use Autocorrect in Outlook to insert common links you often send?
```

Comment:
Hi @ae-cher-ssw 

You don't need to repeat the title in the content

- [ ] 1. Please remove this heading

Reply:
Done